src/app/pages/subjects/subjects.js

```diff
@@ -55,15 +53,13 @@ export default class Subjects extends CMSPageController {
 
         this.bookViewer.filterSubjects(category);
 
-        if (!$.isNode()) {
-            const yTarget = history.state.y;
+        const yTarget = history.state.y;
```

Comment:
hmm, could/should this be `window.history` to clarify that this isn't a local variable?